src/components/LostItemsLink/LostItemsLink.css

```diff
@@ -0,0 +1,5 @@
+.disabledButton {
+  color: #000000;
+  cursor: not-allowed;
+  opacity: 0.65;
+}
```

Comment:
We provide [disabled-button styling](https://folio-org.github.io/stripes-components/?path=/story/components-button--basic-usage) already so I am not confident we need local handling for this, but I am confident that we don't need it twice (I'm lookin' at you, `RequestFeeFineBlockButtons.css`). 

If you believe we _do_ need a local style, (1) please explain why and (2) let's find a place where we can specify it once and share it.

Reply:
Hello @zburke. I will check the component docs again. Indeed, we have handled styling for the disabled button already. However, I don't think we have handled it for `a` tags in. the button component. In my case, since the `Button` component has a `href` or `to` property, the elements have been used as anchor tags rather than buttons.

Reply:
I have created PR regarding to the issue here https://github.com/folio-org/stripes-components/pull/2142